src/Package.php

```diff
@@ -24,4 +53,15 @@ public static function create($descriptor = null, $basePath = null)
 
         return new $packageClass($descriptor, $basePath, true);
     }
+
+  /**
+   * @throws \Exception
+   */
+    private static function isZipPresent()
```

Comment:
Could we throw this exception only if zip support is actually needed?

Reply:
@OriHoch thanks. Yes of course. Do you prefer to have it at a higher level or directly on methods that use the ZIP? Basically during source detection of in the actual Package/ZIP methods?

Reply:
I don't have a preference, whatever makes sense for you